core/templates/dev/head/domain/sidebar/sidebar-status.service.spec.ts

```diff
@@ -16,7 +16,13 @@
  * @fileoverview Tests for SidebarStatusService.
  */
 
-import { SidebarStatusService } from 'domain/sidebar/SidebarStatusService';
+require('domain/sidebar/sidebar-status.service.ts');
```

Comment:
Done

Reply:
Not done

Reply:
I took out the `require` line because I assumed this was added incorrectly. Should it be the other way around: delete the `import` line and restore the `require`?

Reply:
But the line is still there (https://github.com/oppia/oppia/blob/87d98268719463508ed5dae774162ac0de3bd38c/core/templates/dev/head/domain/sidebar/sidebar-status.service.spec.ts)

Reply:
Thanks, I see what you mean now and have corrected it. I will push again.